src/biokbase/narrative/common/kblogging.py

```diff
@@ -116,7 +116,7 @@ def __init__(self, *args):
         handlers.SocketHandler.__init__(self, *args)
         self._dbg = _log.isEnabledFor(logging.DEBUG)
         if self._dbg:
-            _log.debug("Created SocketHandler with args = {}".format(args))
+            _log.debug(f"Created SocketHandler with args = {args}")
```

Comment:
VSCode does auto-conversion of the `format()` things into f-strings, which is quite handy!